pkg/gui/containers_panel.go

```diff
@@ -90,6 +94,37 @@ func (gui *Gui) handleContainerSelect(g *gocui.Gui, v *gocui.View) error {
 	return nil
 }
 
+func (gui *Gui) renderContainerEnv(container *commands.Container) error {
+	mainView := gui.getMainView()
+	mainView.Autoscroll = false
+	mainView.Wrap = gui.Config.UserConfig.Gui.WrapMainPanel
+	envVariablesList := [][]string{}
+	if len(container.Details.Config.Env) > 0 {
+		for _, env := range container.Details.Config.Env {
+			splitEnv := strings.Split(env, "=")
+			// if the value has = in it, lets say export "test=foo=bar" then split will result in the following
+			// {"test", "foo","bar"} hence join all the elements in the slice except the first one to get value
+			envVariablesList = append(envVariablesList,
+				[]string{
+					utils.ColoredString(splitEnv[0]+":", color.FgBlue),
+					utils.ColoredString(strings.Join(splitEnv[1:], "="), color.FgYellow),
+				})
+		}
+	} else {
+		envVariablesList = append(envVariablesList, []string{"Nothing to display"})
```

Comment:
can we internationalise these strings? See `pkg/i18n/english.go` for examples. No need to translate into other languages: just having english versions is sufficient

Reply:
Also seems like there's no need to go through the table approach here. We could just set `renderedTable` directly

Reply:
internationalised the strings and also altered the logic to set `renderedTable`